db/schema.rb

```diff
@@ -95,7 +95,7 @@
     t.string "partner_key"
   end
 
-  create_table "diaper_drive_participants", force: :cascade do |t|
+  create_table "diaper_drive_participants", id: :serial, force: :cascade do |t|
```

Comment:
Can you remove the unrelated schema changes? Sorry about this, I hate when rails does it :(